packages/ui-components/src/__tests__/VaultDetail.test.ts

```
@@ -219,10 +220,10 @@ describe('VaultDetail', () => {
 		});
 
 		await waitFor(async () => {
-			const refreshButton = await screen.findAllByTestId('refresh-button');
-			await userEvent.click(refreshButton[0]);
+			const refreshButton = await screen.getByTestId('top-refresh');
+			await userEvent.click(refreshButton);
 			expect(invalidateQueries).toHaveBeenCalledWith({
-				queryKey: ['100'],
+				queryKey: ['100', QKEY_VAULT + '100'],
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)